packages/diagram/src/module/diagrams/content/uml/classifier/propertiesAndMethods.ts

```diff
@@ -0,0 +1,250 @@
+                        (visibility, scope) = args
+                    `
+                ),
+                jsFun(
+                    (args, context) => {
+                        const scopeFunction = args.getLocalFieldOrUndefined(0)!.value;
+                        if (!(scopeFunction instanceof FunctionObject)) {
+                            throw new Error("scope is not a function");
+                        }
+                        const expressions = scopeFunction.definition.expressions.map(
+                            (expression) => expression.expression
+                        );
+                        const visibility = assertString(context.getField("visibility"));
+                        const scope = context.getField("scope");
+                        const [fields, functions] = convertFieldsAndFunctions(expressions);
+
+                        function addEntriesToScope(entries: string[], index: number): void {
+                            if (entries.length > 0) {
+                                scope.invoke(
+                                    [
+                                        ...entries.map((entry) => ({
+                                            value: new ExecutableConstExpression({
+                                                value: context.newString(visibility + entry)
+                                            })
+                                        })),
+                                        {
+                                            name: "section",
+                                            value: new ExecutableNumberLiteralExpression(undefined, index)
+                                        }
+                                    ],
+                                    context
+                                );
+                            }
+                        }
+                        addEntriesToScope(fields, 0);
+                        addEntriesToScope(functions, 1);
+
+                        return context.null;
+                    },
+                    {
+                        docs: `
+                            Function to take a function in which fields and functions can be declared declaratively.
+                            The content of the function is not executed, but analyzed on the AST level.
+                            Identifiers can be provided both as identifiers and as strings.
+                            A field is defined by an identifier, and optionally a colon and a type identifier.
+                            Example: \`x : int\`
+                            A function is defined by an identifier, an opening and closing bracket with optional parameters,
+                            and optionally a colon and a return type identifier (e.g. test() : int).
+                            Parameters are defined like fields, and separated by commas.
+                            Example: \`point(x : int, y : int) : Point\`
```

Comment:
`use newlines to separate two fields/functions`

Reply:
added (sth similar) with 9c9eecbbf2f389b6d1910ddee2e676b161f6e020